proxy/src/main/java/com/velocitypowered/proxy/protocol/StateRegistry.java

```diff
@@ -118,6 +118,11 @@
                     map(0x47, MINECRAFT_1_12, true),
                     map(0x48, MINECRAFT_1_12_1, true),
                     map(0x4B, MINECRAFT_1_13, true));
+            CLIENTBOUND.register(PlayerListItem.class, PlayerListItem::new,
+                    map(0x38, MINECRAFT_1_8, false),
+                    map(0x2D, MINECRAFT_1_9, false),
+                    map(0x2E, MINECRAFT_1_12_1, false),
```

Comment:
You forgot to specify what this packet's ID for `MINECRAFT_1_12` would be.

Reply:
Fixed.